packages/utilities/psammead-rich-text-transforms/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.0.1 | [PR#3510](https://github.com/bbc/psammead/pull/3510) Add support for inline span tags |
```

Comment:
@hindsc52 2.0.2 here?

Reply:
good spot, copy and paste error. Have updated